### PR TITLE
Update UG and DG for v1.2 done by Siang Ern

### LINKED ISSUE #159: Update UG and DG for v1.2 done by Siang Ern

### PR DESCRIPTION
**Pull request by siangernlow**
Format of the user guide has been changed to improve readability. Features available in the VirusTracker have also been documented in the user guide. User stories and use cases in the developer guide have also been updated.

Close #159

### CODE REVIEW

**Review by hopinxian — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -91,20 +89,63 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 q/true`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 q/false t/criminal`
 
-### Listing: `list`
+#### Adding a visit: 
+
+Adds a visit by the personId, location of visit and date of visit
+
+Format: `add personId location date`
```
Comment (hhdqirui) [inline docs/UserGuide.md:96]: Shouldn't it be "locationId" instead of "location"?

Comment (hopinxian) [reply docs/UserGuide.md:96]: Please refer to it as index rather than id. Reminder that ID is a concept that is hidden from users.

**Review by KohHanMing — APPROVED** [review]
LGTM

**Pull request merged**